### PR TITLE
StAdalbert: Created Location

### PR DESCRIPTION
**Pull request by noxilixon**

### CODE REVIEW

File: group_vars/location_stadalbert/owm.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+location_nice: Tostr. 168
```
Comment (PolynomialDivision) [inline group_vars/location_stadalbert/owm.yml:2]: I think you have a typo here

File: group_vars/location_stadalbert/networks.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    tunnel_wan_gw: 192.168.8.1
+    tunnel_connections: 2
+    tunnel_timeout: 600
+    tunnel_mesh_prefix_ipv4: 10.31.53.24/29
+
+location__channel_assignments_11a_standard__to_merge:
+  stadalbert-core: 40-40
+
+location__channel_assignments_11g_standard__to_merge:
+  stadalbert-core: 5-20
```
Comment (PolynomialDivision) [inline group_vars/location_stadalbert/networks.yml:58]: why 5? Please stick to 1-4-6-9-13

**Comment by noxilixon** [comment]
Danke habs verbessert :)

**Comment by PolynomialDivision** [comment]
Can you rebase on latest master?

**Comment by PolynomialDivision** [comment]
lgtm. thanks

**Pull request merged**